### DISCUSSION TITLE
Next Major Python SDK Version

### DISCUSSION THREAD

**Discussion by saralvasquez** [braintree/braintree_python#152] [Ideas]
## 👋  Welcome
We're using GitHub Discussions as a place to engage with the members of our community, to share ideas, and to support one another.

## 💡  Discussion Topic: Next Major Version of the Braintree Python SDK
The Braintree SDK team is deciding what to include in our next major version release (v5). We are looking for some insight on:
- Language-specific improvements that you would like to see
- Features you'd like us to consider
- Pain points you'd like to see resolutions to

## 🗣 Let Us Know
Please use this thread to voice your opinions, concerns, or thoughts! We'd love to know how we can make the next version better for you.

**Comment by cuu508** [comment]
> Language-specific improvements that you would like to see

Type hints please! There are [type hint stubs in typeshed](https://github.com/python/typeshed/tree/main/stubs/braintree), but many bits are incomplete there. It would be awesome to see first-party type hints.

**Comment by Polandia94** [comment]
Type hints, and some linter/formatter/static check rules to improve code readability
